### PR TITLE
AWS Screen Tutorial

### PR DESCRIPTION
**Pull request by abhijna**
## PR Checklist

- [ ] Release label added
- [ ] PR category label added
- [ ] PR mergeable
- [ ] Clean build logs
- [ ] Linked related issues
- [ ] Colorblindness test https://www.toptal.com/designers/colorfilter/

## PR Description
This is a tutorial that Jose wrote. I edited and packed it for our training website. Some things are not working for me:
- Help commands at the bottom of the cheat 
- Detach and exit command

@josesanchez1815 can you embed the video tutorial when you get a chance?

**Preview link**
https://cfde-training-and-engagement--334.com.readthedocs.build/en/334/Bioinformatics-Skills/Introduction_to_Amazon_Web_Services/introtoaws5_Screen/

## Review format
Please make direct edits or leave comments.

## Timeline
This is for Feb release. Would be good to share this tutorial on Feb 25 at the AWS training workshop

### CODE REVIEW

File: docs/Bioinformatics-Skills/Introduction_to_Amazon_Web_Services/introtoaws5_Screen.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+### Switching between screens
+
+ To switch between the two new screens, i.e. screen 2 and screen 3, type ++ctrl+a+p++. You cannot toggle to the original terminal screen (i.e. screen 1) with this shortcut.
+
+### Detaching screens
+
+ To detach a screen session and return to your original SSH terminal, type ++ctrl+a+d++. You will be taken back to the terminal window with the `echo` command:
+
+![] images-aws/.png
```
Comment (marisalim) [inline docs/Bioinformatics-Skills/Introduction_to_Amazon_Web_Services/introtoaws5_Screen.md]: This image file is missing

Comment (abhijna) [reply docs/Bioinformatics-Skills/Introduction_to_Amazon_Web_Services/introtoaws5_Screen.md]: fixed

**Comment by josesanchez1815** [comment]
Video Embedded at the start of the tutorial

**Comment by marisalim** [comment]
Hi @abhijna @josesanchez1815 - nice tutorial, here are some comments!

I ran AWS Ubuntu Pro 20.04 connected to EC2 Instance Connect web terminal on Firefox.

- [x] since this lesson is nested within the AWS lessons, it should be linked to on the AWS landing pg (move learning objs/time/prereqs there)
- [x]  i'd move this lesson to be before the instance termination steps
- [x]  upload the larger video size from kaltura - 608x402
- [ ] when i returned to the screen with the echo statement, it didn't show the statement anymore. it showed an empty command prompt.
- [x]  at the point when we type `screen -ls` we have 3 screens, but the print out only shows 1 - shouldn't it show 3?
- [x]  sometimes, i needed to type `exit` twice until i got the "screen is terminating" message (but be sure not to `exit` too many times or you exit the entire terminal! -__-)
    ```
    exit
    There are stopped jobs.
  
    exit
    [screen is terminating]
    ```
- [x]  `screen -x <screen name>` seems to work the same as `screen -r <screen name>` - what's the difference? ---> I tested this out and there appears to be no difference, so I took out the -x.
- [x]  `ctrl+a+n` opens a new web browser tab instead of switching to the next screen (but this command works when i SSH to my local Mac terminal). `ctrl+a` 2x worked for switching to a screen - it's hard to tell the direction.
- [x]  can you add that you need to type `ctrl+a` twice to the cheat sheet (it's mentioned in the video but not written)
- [x]  `ctrl+a` didn't open any help info; `screen -h` works
- [x]  `ctrl+a+d` and `ctrl+a` then `ctrl+d` appear to do the same thing for me, both say "[detached from <screen ID>]"
- [x]  Since we have a [Cheat Sheets tab](https://training.nih-cfde.org/en/latest/Cheat-Sheets/), could you move/copy these there too?
- [x]  delete the video link at the very end of the lesson

**Review by s-canchi — COMMENTED** [review]
Hi @abhijna @josesanchez1815 nice tutorial! Here are my comments:

I ran the tutorial on  an EC2 micro instance with Ubuntu Pro 20.04 connected to a web terminal on Safari.

- [x] Explain the concept of multiple screen with an analogy: 
"Think of the screen as opening a new browser window. You can open multiple tabs within the same window using cltr+a+c. You can now toggle between the multiple screens (tabs) using cltr+a+p. You can have commands running in each tab. Detaching the screen allows you to go back to the original window/panel." This is important since you cannot toggle between independent screens created from your original terminal. This is also the reason why one see's only one screen info on detaching even though we created two screens.
- [x] In the reconnect to your screen command, using the first 4 number of the screen name work. i.e. `screen -r 2683`
- [x] It would be useful to add the command that allows you to name the screen such that you can use it to identify when you have multiple sessions instead of numbers. Here is an helpful answer: https://unix.stackexchange.com/questions/12227/setting-a-name-for-a-screen-session/27294
- [x] add a way to delete the screen while in detached state. You can do this by : `screen -S <screen#/name> -X quit`
- [x] Would be useful to add screenshots of the commands across multiple screens.
- [ ] in the video the first few seconds Jose's voice seem to be cut off. 
- [x] agree with @marisalim that the cheat sheets can be moved to the cheat sheets tab.

**Comment by abhijna** [comment]
Thank you, Marisa and Saranya! I made most of the changes. 

I don't know why you can't see the echo statement @marisalim 

> when I returned to the screen with the echo statement, it didn't show the statement anymore. it showed an empty command prompt.

This works fine for me :(. I did add a little blurb to remind users not to clear the screen after they run the echo command. 

@josesanchez1815 can you please address the two comments about the video? 

@s-canchi I was trying to minimize screenshots 😄. Do you think they would add substantially to the user experience? I can add them if so.

**Review by marisalim — APPROVED** [review]
made some edits to clarify screens/tabs. to match the other vidlets on the website, please change to [608x402](https://training.nih-cfde.org/en/latest/CFDE-Internal-Training/Website-Style-Guide/4RecordingStyleGuide/#saving-vidlets).

otherwise, looks ready for a workshop test run!

**Comment by s-canchi** [comment]
If possible  you can just add one screenshot of side by side windows of the screens with the different command but it is not critical. LGTM !

**Review by s-canchi — APPROVED** [review]

**Pull request merged**